GOVERNANCE.md

```diff
@@ -16,11 +16,27 @@ please see [contributor guide] guide.
 Etcd project roles along with their requirements and responsibilities are defined
 in [community membership].
 
-## Decision making process
-
-Decisions are built on consensus between [maintainers] publicly. Proposals and ideas
-can either be submitted for agreement via a GitHub issue or PR, or by sending an email
-to `etcd-maintainers@googlegroups.com`.
+## Decision-making process
+
+Any significant changes affecting the etcd community, including but not limited to
+leadership transitions, governance updates, or project-wide process changes, must
+first be discussed transparently within the sig-etcd leads and current maintainers
+(i.e., approvers of the main etcd repository). These discussions should take place
+through established communication channels such as:
+
+- The bi-weekly sig-etcd meetings
+- The `etcd-maintainers@googlegroups.com` mailing list
+- Private Slack group threads involving sig-etcd leads and maintainers
```

Comment:
Note that group messaging has an 8 member limit enforced by slack (not us) and Kubernetes has very specific rules about private channels in the spirit of openness:
https://www.kubernetes.dev/docs/comms/slack/#private-channels

Reply:
thx for the reminder. 

It's just one of the options. Also current we don't exceed 8 members :)